base/java-tools/src/com/netscape/cmstools/system/SecurityDomainHostCLI.java

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright Red Hat, Inc.
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+//
+package com.netscape.cmstools.system;
+
+import org.dogtagpki.cli.CLI;
+
+import com.netscape.certsrv.system.SecurityDomainClient;
+import com.netscape.certsrv.system.SecurityDomainHost;
+
+/**
+ * @author Endi S. Dewata
+ */
+public class SecurityDomainHostCLI extends CLI {
+
+    public SecurityDomainCLI parent;
+
+    public SecurityDomainHostCLI(SecurityDomainCLI parent) {
+        super("host", "Security domain host management commands", parent);
+        this.parent = parent;
+
+        addModule(new SecurityDomainHostFindCLI(this));
+        addModule(new SecurityDomainHostShowCLI(this));
+        addModule(new SecurityDomainHostAddCLI(this));
+        addModule(new SecurityDomainHostRemoveCLI(this));
+    }
+
+    public SecurityDomainClient getSecurityDomainClient() throws Exception {
+        return parent.getSecurityDomainClient();
+    }
+
+    public static void printSecurityDomainHost(SecurityDomainHost host) {
+
+        System.out.println("  Host ID: " + host.getId());
+        System.out.println("  Hostname: " + host.getHostname());
+        System.out.println("  Port: " + host.getPort());
+        System.out.println("  Secure Port: " + host.getSecurePort());
+
```

Comment:
For the sake of completeness, I feel that we should display the following fields too:
````
adminPort
agentPort
clone
eePort
unsecurePort
````

Reply:
I have added the `clone`. The `unsecurePort` is the same as `port` which is already displayed. The `eePort`, `adminPort`, `agentPort` are legacy ports. They used to be different ports, but now we only use a single `securePort` for all of them which is already displayed too. We eventually drop will those legacy ports completely. I've changed the CLI to accept a single `securePort` parameter and set the legacy ports to the same value.